src/binning_based/rectangular/TransferOperator.jl

```diff
@@ -0,0 +1,383 @@
+using GroupSlices, DelayEmbeddings, SparseArrays
+
+export TransferOperator, 
+    transferoperator, TransferOperatorApproximationRectangular,
+    InvariantMeasureEstimate, invariantmeasure,
+    binhist
+
+"""
+    TransferOperator(r::RectangularBinning)
+
+A probability estimator based on binning data into rectangular boxes dictated by 
+the binning scheme `r`, then estimating the transfer (Perron-Frobenius) operator 
+over the bins. Assumes that the input data are sequential.
+
+The transfer operator ``P^{N}``is computed as an `N`-by-`N` matrix of transition 
+probabilities between the states defined by the partition elements, where `N` is the 
+number of boxes in the partition that is visited by the orbit/points. The left invariant
+distribution ``\\rho^{N}``, where ``\\rho^{N}P^{N} = \\rho^{N}``, approximates the 
+invariant density of the system subject to the partition `r`. Thus, ``\\rho^{N}`` can 
+be taken as a probability distribution over the partition elements.
+
+This implementation follows the approach in Diego et al. (2019)[^Diego2019].
+
+See also: [`RectangularBinning`](@ref).
+
```

Comment:
The problem here is that the docstring doesn't *actually* tell you how to obtain ρ. Pernaps some cross-references are missing?